pkgs/os-specific/darwin/utm/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, undmg
+, fetchurl
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "utm";
+  version = "3.2.4";
+
+  src = fetchurl {
+    url = "https://github.com/utmapp/UTM/releases/download/v${version}/UTM.dmg";
+    sha256 = "sha256-ejUfL6UHqMusVfaglGlODKtFfKbNwzZ1LmRkcSzieso=";
+  };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = ".";
+  installPhase = ''
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications
+  '';
+
+  meta = with lib; rec {
+    description = "UTM is a full featured system emulator and virtual machine host for iOS and macOS";
```

```suggestion
    description = "Full featured system emulator and virtual machine host for iOS and macOS";
```

Short `description` should never be "self-citing".